gems/quilt_rails/test/test_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+require 'minitest/autorun'
+require 'rails'
+require 'mocha/minitest'
+require 'quilt_rails'
+
+module Dummy
```

Comment:
Do you need the dummy app?

Reply:
I guess not, I am not really sure what it's for TBH, copied the test boilerplate from sewing_kit as well.